libs/core/src/components/pds-tabs/docs/pds-tabs.mdx

```diff
@@ -3,62 +3,64 @@ import { components } from '../../../../dist/docs.json';
+For best practices in accessibility, be sure to provide a `tablist-label` and a unique `component-id`. The `tabslist-label` will provide context to screen readers about what kind of tabslist information is being presented.
 
-In order for all accessibility to properly compile, you must provide a few required properties to the component.
+## Properties
 
-**1. A descriptive `tablist-label`:** Provides context to screen readers about what kind of tablist information is being presented
+<DocArgsTable componentName='pds-tabs' docSource={components} />
 
-**2. A unique `component-id`:** Provides a unique `id` to the tabs component, be sure it doesn't match any other `id`'s on the page
+### Tabs
 
-## Properties
+<DocArgsTable componentName='pds-tab' docSource={components} />
 
-<DocArgsTable componentName='pds-tabs' docSource={components} />
+### Tabpanel
+
+<DocArgsTable componentName='pds-tabpanel' docSource={components} />
 
 ## Variant
 
-### Default (Primary)
+### Primary
 
-Default tabs styles
+The default style of Tabs.
 
 <DocCanvas client:only mdxSource={{
   react: `
-<PdsTabs activeTabName="Dollop" variant="primary" componentId="primary" tablistLabel="Foo">
-  <PdsTab name="Sturdy">Sturdy</PdsTab>
-  <PdsTab name="Dollop">Dollop</PdsTab>
-  <PdsTab name="Waffle">Waffle</PdsTab>
-  <PdsTabpanel name="Sturdy">Content Sturdy</PdsTabpanel>
-  <PdsTabpanel name="Dollop">Content Dollop</PdsTabpanel>
-  <PdsTabpanel name="Waffle">Content Waffle</PdsTabpanel>
-</PdsTabs>
+  <PdsTabs activeTabName="Dollop" variant="primary" componentId="primary" tablistLabel="Foo">
+    <PdsTab name="Sturdy">Sturdy</PdsTab>
+    <PdsTab name="Dollop">Dollop</PdsTab>
+    <PdsTab name="Waffle">Waffle</PdsTab>
+    <PdsTabpanel name="Sturdy">Content Sturdy</PdsTabpanel>
+    <PdsTabpanel name="Dollop">Content Dollop</PdsTabpanel>
+    <PdsTabpanel name="Waffle">Content Waffle</PdsTabpanel>
+  </PdsTabs>
 `,
   webComponent: `
-<pds-tabs active-tab-name="Dollop" variant="primary" component-id="primary" tablist-label="Foo">
-  <pds-tab name="Sturdy">Sturdy</pds-tab>
-  <pds-tab name="Dollop">Dollop</pds-tab>
-  <pds-tab name="Waffle">Waffle</pds-tab>
-  <pds-tabpanel name="Sturdy">Content Sturdy</pds-tabpanel>
-  <pds-tabpanel name="Dollop">Content Dollop</pds-tabpanel>
-  <pds-tabpanel name="Waffle">Content Waffle</pds-tabpanel>
-</pds-tabs>
+  <pds-tabs active-tab-name="Dollop" variant="primary" component-id="primary" tablist-label="Foo">
+    <pds-tab name="Sturdy">Sturdy</pds-tab>
+    <pds-tab name="Dollop">Dollop</pds-tab>
+    <pds-tab name="Waffle">Waffle</pds-tab>
+    <pds-tabpanel name="Sturdy">Content Sturdy</pds-tabpanel>
+    <pds-tabpanel name="Dollop">Content Dollop</pds-tabpanel>
+    <pds-tabpanel name="Waffle">Content Waffle</pds-tabpanel>
+  </pds-tabs>
```

Comment:
I'll fix this, but I wanted to bring it to your attention. When making this change it adds space in the code example. This occurs due to how backtick works.

Reply:
Ahhh thanks for catching that. Didn't think that would have an effect.